packages/psss-server/src/types.ts

```diff
@@ -22,6 +24,53 @@ export interface PsssRequest<>extends Request<ParamsDictionary, unknown, PsssReq
 }
 
 export interface PsssResponseBody {
-  accessPolicy?: {};
+  accessPolicy?: AccessPolicyObject;
   error?: string;
 }
+
+export interface Credentials {
+  emailAddress: string;
+  password: string;
+}
+
+export type QueryParameters = Record<string, string>;
+
+export type RequestBody = Record<string, unknown>;
+
+interface FetchHeaders {
+  Authorization: string;
+  'Content-Type'?: string;
+}
+
+export interface FetchConfig {
+  method: string;
+  headers: FetchHeaders;
+  body?: string;
+}
+
+interface AuthResponseUser {
+  email: string;
+  accessPolicy: AccessPolicyObject;
+}
+
+export interface AuthResponse {
+  accessToken?: string;
+  refreshToken?: string;
+  user?: AuthResponseUser;
+}
+
+export interface SessionDetails {
+  email: string;
+  accessPolicy: AccessPolicyObject;
+  accessToken: string;
+  refreshToken: string;
+}
+
+export interface SessionColumns {
```

Comment:
We have named a similar type in indicators `IndicatorFields`, may be a good idea to rename to `SessionFields`

Reply:
Very happy to!